packages/@aws-cdk/aws-eks/test/alb-controller.test.ts

```diff
@@ -1,28 +1,10 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import { SynthUtils } from '@aws-cdk/assert-internal';
 import * as iam from '@aws-cdk/aws-iam';
 import '@aws-cdk/assert-internal/jest';
-import { Cluster, KubernetesVersion, AlbController, AlbControllerVersion } from '../lib';
+import { Cluster, KubernetesVersion, AlbController, AlbControllerVersion, HelmChart } from '../lib';
 import { testFixture } from './util';
 
-test('minimal snapshot', () => {
```

Comment:
This test is basically a duplicate of the integ: https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-eks/test/integ.alb-controller.ts

Not needed.